### PR TITLE
Add an issue template override to all bridged providers

### PR DESCRIPTION
**Pull request by guineveresaenger**
This PR adds an `ISSUE_TEMPLATE/bug.yaml` to each bridged provider repo that will override the [org-wide default](https://github.com/pulumi/.github/blob/main/.github/ISSUE_TEMPLATE/bug.yaml).

Of note:

- Asks for resources affected
- Asks for console output snippet
- Asks for repro and explains why it's needed
- Asks for logs

Concerns I'd like the reviewer to consider:

- Is this the log output we'll want? Do we want GRPC-level logs instead?
- Should we acknowledge that sometimes a complete repro we can run on our end may be impossible?
- Any wording tweaks

Fixes https://github.com/pulumi/platform-providers-team/issues/157.

### CODE REVIEW

File: provider-ci/internal/pkg/templates/bridged-provider/.github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    attributes:
+      label: Affected Resource(s)
+      description: Please list the affected Pulumi Resource(s) or Function(s).
+    validations:
+      required: false
+  - type: textarea
+    id: versions
+    attributes:
+      label: Output of `pulumi about`
+      description: Provide the output of `pulumi about` from the root of your project.
```
[the remark this answers is not in the document]
Comment (VenelinMartinov) [reply provider-ci/internal/pkg/templates/bridged-provider/.github/ISSUE_TEMPLATE/bug.yaml:41]: `<details>`?

File: provider-ci/internal/pkg/templates/bridged-provider/.github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      description: Please summarize what happened, including what Pulumi commands you ran, as well as
+        an inline snippet of any relevant error or console output.
+    validations:
+      required: true
+  - type: textarea
+    id: reproducible
+    attributes:
+      label: Sample program and log output
+      description: |
+        Please provide us with a minimal, self-contained Pulumi program that reproduces this behavior so that
```
[the remark this answers is not in the document]
Comment (VenelinMartinov) [reply provider-ci/internal/pkg/templates/bridged-provider/.github/ISSUE_TEMPLATE/bug.yaml:23]: We could have docs be a separate issue type?

File: provider-ci/internal/pkg/templates/bridged-provider/.github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        we can investigate on our end. Without a functional reproduction, we will not be able to prioritize this bug.
+        If relevant, please also provide us with the output of 
+        `pulumi up --logtostderr --logflow -v=10` from the root of your project.
```
[the remark this answers is not in the document]
Comment (t0yv0) [reply provider-ci/internal/pkg/templates/bridged-provider/.github/ISSUE_TEMPLATE/bug.yaml:26]: We could make sure GRPC logs are written as is to the standard logs. They are currently written conceptually but in a harder to parse format.

**Comment by iwahbe** [comment]
@guineveresaenger We have all offered suggestions (myself included), and we can continue to iterate on the template. I think what you wrote is an improvement to what we have now. I'd love it if we could merge and then iterate, instead of vice versa.

**Pull request merged**